lib/collection/src/update_handler.rs

```diff
@@ -329,7 +345,12 @@ impl UpdateHandler {
                             ) {
                                 // Perform some actions when optimization if finished
                                 Ok(result) => {
-                                    tracker_handle.update(TrackerStatus::Done);
+                                    if result == false {
+                                        num_points_optimized = 0;
+                                    }
+
+                                    tracker_handle
+                                        .update(TrackerStatus::Done, Some(num_points_optimized));
```

Comment:
Alternatively, we can just return num points optimized from the `optimize()` function instead of a bool.

Reply:
Did this.